src/main/java/org/radarcns/security/filter/AuthenticationFilter.java

```diff
@@ -47,6 +50,14 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         if (token == null) {
             res.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
             res.setHeader("WWW-Authenticate", "Bearer");
+            res.setHeader("Error", "No Token Provided!");
+            String jsonMsg = SecurityUtils.getJsonError("Please provide a valid token" +
+                            " in the authentication header",
+                    new TokenException("No token was provided " +
+                            "with the request and thus the request " +
+                            "cannot be authorized")).toString();
```

Comment:
I think better to sent 401 `Unauthorized access exception` for this than `TokenException`.